server/journeys/referral/on-behalf-of/controller.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from 'express'
+
+export class ReferralOnBehalfOfController {
```

Comment:
since we don't have `Referral` in the route name, maybe this can just be `OnBehalfOfController` as well?

Reply:
It is part of the overall Referral journey so it makes sense to me to have it in the name

Reply:
yep. i'm fine with either:
```
OnBehalfOfController
OnBehalfOfRoutes
```
or:
```
ReferralOnBehalfOfController
ReferralOnBehalfOfRoutes
```